x/rollapp/types/message_create_rollapp.go

```diff
@@ -45,45 +44,31 @@ func (msg *MsgCreateRollapp) GetSignBytes() []byte {
 	return sdk.MustSortJSON(bz)
 }
 
-func (msg *MsgCreateRollapp) ValidateBasic() error {
-	_, err := sdk.AccAddressFromBech32(msg.Creator)
-	if err != nil {
-		return sdkerrors.Wrapf(ErrInvalidAddress, "invalid creator address (%s)", err)
+func (msg *MsgCreateRollapp) GetRollapp() Rollapp {
+	// Build the genesis state from the genesis accounts
+	var rollappGenesisState *RollappGenesisState
```

Comment:
if genesis state remains `nil` I think we would get an error in the genesis event later on:
https://github.com/dymensionxyz/dymension/blob/main/x/rollapp/keeper/keeper.go#L68
Not sure if that's expected, if not, we should instantiate `rollappGenesisState` regardless of genesis accounts being empty or not. 
Unless it's expected that for some reason we enforce genesis accounts not being empty. In that case, we should probably fail on validation or something.